src/bot.js

```diff
@@ -117,3 +106,19 @@ export async function initBot() {
     });
   }
 }
+
+// Connect to the database
+async function connectWithRetry(MONGODB) {
+  return mongoose
+    .connect(MONGODB, {
+      useNewUrlParser: true,
+      useUnifiedTopology: true,
+    })
+    .then((_) => {
```

Comment:
Do you need the underscore here? You could get away with omitting it.

Reply:
The _ was already there. But I guess you are right

Reply:
Fixed